WordPress/gutenberg
# [Mobile] Improve keyboard hide button

## Description

This PR is related with https://github.com/wordpress-mobile/gutenberg-mobile/pull/480. Changes are made to allow passing style to toolbar container. Also updated the border style of the toolbar container due to https://github.com/wordpress-mobile/gutenberg-mobile/pull/480#issuecomment-455620000

## How has this been tested?
Mobile side is tested with steps in https://github.com/wordpress-mobile/gutenberg-mobile/pull/48

Web test:
- Run `npm install` `npm run dev` `docker-compose up`
- Open http://localhost:8888/wp-admin
- Verify there's no anomaly in toolbars

## Checklist:
